stacks-signer/src/config.rs

```diff
@@ -151,6 +151,9 @@ pub struct SignerConfig {
     pub max_tx_fee_ustx: Option<u64>,
     /// The path to the signer's database file
     pub db_path: PathBuf,
+    /// How much time between the first block proposal in a tenure and the next bitcoin block
+    ///  must pass before a subsequent miner isn't allowed to reorg the tenure
```

Comment:
same comment about "must pass"

Reply:
Yep, updated in [62aac67](https://github.com/stacks-network/stacks-core/pull/4956/commits/62aac67851b4104d2f12a549afaa00bbc82436c6)